style/tailwind.css

```diff
@@ -1,3 +1,141 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.markdown-content {
+  @apply bg-base-200 mt-4 p-5 rounded space-y-4;
+}
+
+.markdown-content h1 {
+  @apply text-2xl font-black;
+}
+
+.markdown-content h2 {
+  @apply text-2xl font-bold;
+}
+
+.markdown-content h3 {
+  @apply text-xl font-black;
+}
```

Comment:
Trying to do the styling with tailwind classes got pretty ugly since I can't add the classes directly to the elements rendered from the markdown, so I did the styling here while using tailwind classes for everything that wasn't a one-off style.

Reply:
No probs, ya since you're using tailwind classes anyway then we'll probably be fine.